src/editor/editorui.js

```diff
@@ -79,10 +97,40 @@ export default class EditorUI {
 	 */
 	destroy() {
 		this.stopListening();
-		this.view.destroy();
+
 		this.focusTracker.destroy();
+
+		this._editableElements = new Map();
+	}
+
+	/**
+	 * Returns the editable editor element with the given name or null if editable does not exist.
+	 *
+	 * @param {String} [rootName=main] The editable name.
+	 * @returns {HTMLElement|null}
+	 */
+	getEditableElement( rootName = 'main' ) {
+		return this._editableElements.has( rootName ) ? this._editableElements.get( rootName ) : null;
```

Comment:
```suggestion
		return this._editableElements.get( rootName );
```

Reply:
The difference is that `get()` returns `undefined` (instead of `null`) if element doesn't exists, but I assume it doesn't matter that much in this case. Alternatively one can use `return this._editableElements.get( rootName ) || null;` which is shorter.